documentapi/src/test/java/com/yahoo/documentapi/messagebus/protocol/test/PolicyTestCase.java

```diff
@@ -369,7 +369,7 @@ public void multipleGetRepliesAreMergedToFoundDocument() {
         frame.setHop(new HopSpec("test", getDocumentRouteSelectorRawConfig())
                 .addRecipient("foo").addRecipient("bar"));
         frame.setMessage(new GetDocumentMessage(new DocumentId("id:ns:testdoc::yarn"), "[all]"));
-        List<RoutingNode> selected = frame.select(2);
+        List<RoutingNode> selected = frame.select(1);
```

Comment:
Instead of updating the expectations, consider updating `getDocumentRouteSelectorRawConfig()` to return a config where both routes use a selection with `testdoc` so that merging can be explicitly tested. Otherwise I think this test can be removed entirely since we already test selection-based routing of Gets in other tests (`get_document_messages_are_sent_to_the_route_handling_the_given_document_type`)

Reply:
Fixed

Reply:
Fixed